### PR TITLE
Add source column to merged context CSVs

### LINKED ISSUE #51: Include source(s) in merged context
It would be helpful for debugging if in the merged contexts, there were a fifth column that kept track of from which simple context(s) each record was derived

### PR DESCRIPTION
**Pull request by cthoyt**
Closes #51 

This PR adds an additional field to the `PrefixExpansion` class that can be optionally annotated with the source from which the class was instantiated. This is then used during the `Context.combine()` function such that when the `merged` and `merged.oak` contexts are generated, each record gets annotated with the simple context from which it came.

This PR also runs the ETL pipeline to regenerate the merged prefix maps, now with the sources annotated in the merged files. The new `source` column does not affect reading in any way.

Interestingly, this sheds some light on #49 - now we know that the `WIKIDATA` prefix comes from the Bioregistry and `wd` from Prefix.cc.

### CODE REVIEW

File: src/prefixmaps/data/merged.csv [hunk 1]
```diff
@@ -1,4297 +1,4299 @@
...
-merged,wds,http://www.wikidata.org/entity/statement/,canonical
-merged,wdt,http://www.wikidata.org/prop/direct/,canonical
-merged,wdtn,http://www.wikidata.org/prop/direct-normalized/,canonical
-merged,wdv,http://www.wikidata.org/value/,canonical
-merged,web,http://www.w3.org/2007/uwa/context/web.owl#,canonical
-merged,webac,http://fedora.info/definitions/v4/webac#,canonical
-merged,webbox,http://webbox.ecs.soton.ac.uk/ns#,canonical
-merged,WEBELEMENTS,https://www.webelements.com/,canonical
-merged,webservice,http://www.openlinksw.com/ontology/webservices#,canonical
-merged,webtlab,http://webtlab.it.uc3m.es/,can
```
Comment (cthoyt) [inline src/prefixmaps/data/merged.csv:4179]: See here - `WIKIDATA` comes from the Bioregistry

Comment (cthoyt) [inline src/prefixmaps/data/merged.csv:4146]: See here - `wd` comes from Prefix.cc

Comment (sierra-moxon) [reply src/prefixmaps/data/merged.csv:4179]: The prefix in bioregistry is `wikidata`, not `WIKIDATA` right?

Comment (cthoyt) [reply src/prefixmaps/data/merged.csv:4179]: correct, but the processing in this repo uppercases the bioregistry prefixes

Comment (sierra-moxon) [reply src/prefixmaps/data/merged.csv:4179]: right; so should the source be prefixmaps?

Comment (cthoyt) [reply src/prefixmaps/data/merged.csv:4574]: I don't consider prefixmaps to be its own registry, this is just a transform on a Bioregistry prefix. Given #48 and https://github.com/biopragmatics/bioregistry/pull/969, it should be more obvious how all uppercase prefix synonyms get in

Comment (cthoyt) [reply src/prefixmaps/data/merged.csv:4574]: @sierra-moxon so to try and explain again, the point of the source column is to help track down where a given expansion came from. In this case, even though a transformation was done on it, it's still from the Bioregistry.

**Comment by cthoyt** [comment]
@sierra-moxon please let me know if there's anything you'd like changed here / what the way forward is

File: src/prefixmaps/data/merged.csv (hunk 1 above)
Comment (sierra-moxon) [reply src/prefixmaps/data/merged.csv:4574]: how about we call it "expansion_source" ?

**Review by sierra-moxon — APPROVED** [review]

**Comment by cthoyt** [comment]
@sierra-moxon can we please merge this? It will also solve @glass-ships' problem in #55

File: src/prefixmaps/data/merged.csv (hunk 1 above)
Comment (sierra-moxon) [reply src/prefixmaps/data/merged.csv:4574]: @cthoyt - did you see that this was "approved" and had this question for you?  Please merge as you see fit, but consider being more explicit in the naming of this column.

Comment (cthoyt) [reply src/prefixmaps/data/merged.csv:4574]: I did not see this, thank you for the ping. I will update that as you suggested :)

**Pull request merged**